MsCorePkg/MacAddressEmulationDxe/MacAddressEmulationDxe.c

```diff
@@ -0,0 +1,278 @@
+/** @file
+
+  DXE Driver for handling MAC Address Emulation.
+
+  Copyright (C) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+#include "MacAddressEmulationDxe.h"
+
+#include <Library/BaseLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/DebugLib.h>
+#include <Library/UefiLib.h>
+#include <Library/UefiBootServicesTableLib.h>
+#include <Library/NetLib.h>
+
+#include <Library/MacAddressEmulationPlatformLib.h>
+
+/**

```

Comment:
Can you clean up the documentation format to match the normal conventions? Such as drop `@brief`.

Not a big deal for existing code but there's no reason to mix styles in new code being added.